recipes/default.rb

```diff
@@ -117,7 +127,7 @@
 execute 'Force sync hardware clock with system clock' do
   command 'hwclock --systohc'
   action :run
-  only_if { node['ntp']['sync_hw_clock'] && !(platform_family?('windows') || platform_family?('freebsd')) }
+  only_if { node['ntp']['sync_hw_clock'] && !(platform_family?('windows') || platform_family?('solaris2') || platform_family?('freebsd')) }
```

Comment:
platform_family? takes comma separated list of platforms so you can considate the solaris2 and freebsd check here. platform_family?('solaris2', 'freebsd')